### PR TITLE
Fix formatting, update to latest runtime, Quetoo and modules

### PR DESCRIPTION
**Pull request by AsciiWolf**

### CODE REVIEW

File: org.quetoo.Quetoo.appdata.xml [hunk 1]
```diff
@@ -43,6 +43,6 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
-    <release version="0.1.0" date="2021-04-08" type="development"/>
+    <release version="0.1.0" date="2022-04-08" type="development"/>
```
Comment (AsciiWolf) [inline org.quetoo.Quetoo.appdata.xml:35]: The release date is based on the date of the last commit. Not sure about the version number though.

**Comment by bbhtt** [comment]
you need the appdata patch from the other PR.

also can you keep the whitespace changes in a seperate commit? Helps when reviewing by commit.

**Comment by bbhtt** [comment]
is there any reason to update the app and a dependency? I'd like to avoid them if possible since this is already unmaintained.

**Comment by AsciiWolf** [comment]
> you need the appdata patch from the other PR.
> 
> also can you keep the whitespace changes in a seperate commit? Helps when reviewing by commit.

I have updated the AppStream metadata (and fixed the second screenshot that was duplicated in your PR).

Sorry, but I won't keep the whitespace changes in separate commit since it would require me to redo all the changes I already did and also it would bring unnecessary complexity. If you have more time, feel free to do it in your PR.

**Comment by AsciiWolf** [comment]
> is there any reason to update the app and a dependency? I'd like to avoid them if possible since this is already unmaintained.

There are actually two reasons. The first one is the "new version dialog" that appears every time the game is launched. The second one are graphical glitches that I have in-game with the old version and I want to test whether there are still the same glitches with more recent (latest "master" - so basically a stable beta since the actual development happens in "develop") Quetoo (and dependencies) version.

**Comment by AsciiWolf** [comment]
So, the graphical glitches are sadly still there.

screenshot

Just like with your PR.

The "new version available" dialog is also still there. And there are no online dedicated servers displayed, although I am not sure whether Quetoo supports them.

**Comment by bbhtt** [comment]
Ok cool, I'll apply your other screenshot fix in my PR and update it.

I'd like the keep the diff minimum (it also allows blame to work later)

**Pull request closed without merging**